tensorflow_recommenders_addons/dynamic_embedding/core/kernels/cuckoo_hashtable_op.cc

```diff
@@ -304,6 +304,18 @@ class CuckooHashTableOfTensors final : public LookupInterface {
     return table_->export_values(ctx, value_dim);
   }
 
+  Status Save(OpKernelContext* ctx, const string filepath,
```

Comment:
SaveToFile might be better for possible extending in the future.

Reply:
Accept.